clusters/hive/pools/msp/admins_msp-cluster-pool_rbac.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+items:
+- apiVersion: v1
+  kind: Namespace
+  metadata:
+    name: msp-cluster-pool
```

Comment:
"pools" could be a reasonable future-proofing? Sounds like a nit but it may incorrectly make someone think there should be only one pool in the namespace

Reply:
Hi @petr-muller that's a good point and I like it. So, I have changed it in all affected yaml files and also updated it in Vault.